nox/sessions.py

```diff
@@ -350,7 +350,9 @@ def run(
             print("Current Git commit is", out.strip())
 
         :param env: A dictionary of environment variables to expose to the
-            command. By default, all environment variables are passed.
+            command. By default, all environment variables are passed. You
+            can block an environment variable from the outer environment by
+            setting it to None.
```

Comment:
Docs update here.

Reply:
Ah, I do see it in https://nox.thea.codes/en/stable/config.html#nox.sessions.Session.run. Sorry about the noise.